code/_helpers/game.dm

```diff
@@ -281,15 +281,15 @@
 
 	//A list of every mob with a client
 	for(var/mob in player_list)
-		if(isnull(mob)) //VOREStation Add - Fix for a weird bug with nulls in this list
-			player_list -= mob //VOREStation Add
-			continue //VOREStation Add
+		if(!istype(mob, /mob))
+			crash_with("There is a null or non-mob reference inside player_list.")

```

Comment:
Yes, replaced with the version Polaris put in.  Altho on a second look each had advantages. I think I will revise to be best of both.